src/main/java/org/maxgamer/quickshop/Command/SubCommands/SubCommand_SilentUnlimited.java

```diff
@@ -36,7 +36,7 @@ public void onCommand(@NotNull CommandSender sender, @NotNull String commandLabe
             shop.update();
             MsgUtil.sendControlPanelInfo(sender, shop);
             sender.sendMessage(MsgUtil.getMessage("command.toggle-unlimited", sender,
-                    (shop.isUnlimited() ? plugin.getConfig().getString("command.toggle-unlimited.unlimited") : plugin.getConfig().getString("command.toggle-unlimited.limited"))));
+                    (shop.isUnlimited() ? plugin.getConfig().getString("shop.toggle-unlimited.unlimited") : plugin.getConfig().getString("shop.toggle-unlimited.limited"))));
```

Comment:
All language phrase should use MsgUtil and store in messages.yml
Config only for config, don't save any phrases.